tsconfig.json

```diff
@@ -1,68 +1,46 @@
 {
-  "extends": "@tsconfig/node16/tsconfig.json",
-  // See: https://github.com/tsconfig/bases/blob/main/bases/node16.json
+  "extends": "@tsconfig/node16-strictest/tsconfig.json",
+  // See: https://github.com/tsconfig/bases/blob/main/bases/node16-strictest.combined.json
```

Comment:
Is this list going to go out of date quickly? Probably not a big deal but if we're going to enumerate all the options in comments I feel like you could just have the options there without extending anything 😛

Reply:
We could do that, but the idea here is that we're following a good strict base from upstream.

Inlining the whole json here is probably overkill, maybe we can just remove the `Inherited Config` section, and folks can just `ctrl + click` the url?